hyrax.gemspec

```diff
@@ -49,7 +49,7 @@ SUMMARY
   spec.add_dependency 'flot-rails', '~> 0.0.6'
   spec.add_dependency 'font-awesome-rails', '~> 4.2'
   spec.add_dependency 'hydra-derivatives', '~> 3.3'
-  spec.add_dependency 'hydra-editor', '>= 3.3', '< 5.0'
+  spec.add_dependency 'hydra-editor', '>= 3.0', '~> 5.0'
```

Comment:
Does this allow a 3.x hydra-editor?  I would think that the `~> 5.0` would force to only 5.x releases.  Maybe `< 6` would work?

Reply:
The idea was to allow for 3.x yes so I've made that change. Do you think the change that removes to_model from the WorkflowResponsiblity would be problematic?

Reply:
Ideally I'd like to force the upgrade on people ;)

Reply:
Tests still passing with that change @cjcolvar